<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="currentUserLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Current User:</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentUserPuzzleRatingLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Puzzle Rating:</string>
   </property>
  </widget>
  <widget class="QLabel" name="filtersLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Filters:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="openingRadioButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>opening</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="midgameRadioButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>midgame</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="endgameRadioButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>endgame</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="forkRadioButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>fork</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="pinRadio">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>pin</string>
   </property>
  </widget>
  <widget class="QLabel" name="positionLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Position:</string>
   </property>
  </widget>
  <widget class="QLabel" name="motifLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Motif:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="tradeLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>trade</string>
   </property>
  </widget>
  <widget class="QLabel" name="taskLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Task:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="winRadioButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>win</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="drawRadioButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>draw</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="defendRadioButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>defend</string>
   </property>
  </widget>
  <widget class="QLabel" name="colourLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Colour:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="whiteRadioButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>white</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="blackRadioButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>black</string>
   </property>
  </widget>
  <widget class="QSlider" name="ratingSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="minimum">
    <number>400</number>
   </property>
   <property name="maximum">
    <number>2800</number>
   </property>
   <property name="sliderPosition">
    <number>800</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="tickInterval">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="ratingLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Rating:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
   <property name="sliderPosition">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="tickInterval">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="numOfPuzzlesSlider">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Number of Puzzles:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="getPuzzlesButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Get Puzzles</string>
   </property>
  </widget>
  <widget class="QLabel" name="dailyPuzzleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Daily Puzzle:</string>
   </property>
  </widget>
  <widget class="QLabel" name="matchupLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Niemman Vs Carlsen</string>
   </property>
  </widget>
  <widget class="QLabel" name="dailyRatingLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Rating:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dailyPuzzleButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Daily Puzzle</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
